docs/pages/database-access/guides/clickhouse-self-hosted.mdx

```diff
@@ -0,0 +1,318 @@
+---
+title: Database Access with ClickHouse
+description: How to configure Teleport database access with ClickHouse.
+---
+
+The Teleport Clickhouse integration allows you to enroll ClickHouse databases
+with Teleport.
+
+The Teleport Database Service authenticates to ClickHouse using x509
+certificates, which are available for the ClickHouse HTTP and native TCP
+[interfaces](https://clickhouse.com/docs/en/interfaces/overview). The Teleport
+Database Service can communicate in both the ClickHouse TCP and HTTP protocols,
+and you can select which protocol to use when configuring the Teleport Database
+Service.
+
+Teleport audit logs for query activity are only supported for the ClickHouse
+HTTP interface. Teleport support for ClickHouse's native interfaces does not
+include audit logs for database query activity.
+
+<Admonition type="warning" title="Preview">
+
+Database access for ClickHouse Database is currently in Preview mode.
+
+</Admonition>
+
+This guide will help you to:
+
+- Install and configure a Teleport database agent.
+- Set up Teleport to access your self-hosted ClickHouse database.
+- Connect to your database through Teleport.
+
+<ScopedBlock scope={["oss", "enterprise"]}>
+![Teleport Database Access Self-hosted ClickHouse](../../../img/database-access/guides/clickhouse_selfhosted_selfhosted.png)
+</ScopedBlock>
+<ScopedBlock scope={["cloud"]}>
+![Teleport Database Access ClickHouse Cloud](../../../img/database-access/guides/clickhouse_selfhosted_cloud.png)
+</ScopedBlock>
+
+## Prerequisites
+
+(!docs/pages/includes/commercial-prereqs-tabs.mdx!)
```

Comment:
This page's scopes include OSS, but we're using the commercial prereqs partial which suggests that enterprise bins of the local tools are required.

Reply:
Thanks, It should be (!docs/pages/includes/edition-prereqs-tabs.mdx!) Fixed.